### ISSUE TITLE
regitlint ignores profile

### ISSUE THREAD

**Issue by flaviolazzarini** [sethreno/ReGitLint#7]
Hi and thank you for this great tool. I've made a custom codecleanup profile and I try to use it with the parameter `--jb --profile=custom_profile`.

Unfortunately the cli is ignoring the parameter and uses the Built In config. Is it a bug, or am I doing something wrong?

**Comment by bart-degreed** [comment]
I'm running into the same issue and it looks like a bug indeed. The check [here](https://github.com/sethreno/ReGitLint/blob/master/ReGitLint/Cleanup.cs#L268) does not properly recognize custom profiles. Running with `--print-command` option reveals the issue:

```
dotnet regitlint --print-command --jb --profile=custom
```

produces:

```
dotnet tool run jb cleanupcode "c:\Projects\demo.sln" --include=";**/*" --profile=custom --profile="Built-in: Full Cleanup" -dsl=GlobalAll -dsl=SolutionPersonal -dsl=ProjectPersonal
```

This results in cleanupcode to discard the first profile and use the second one. The workaround I found is to trick the code by passing `--profile` too:

```
dotnet regitlint --print-command --jb --profile --jb --profile=custom
```

produces:

```
dotnet tool run jb cleanupcode "c:\Projects\demo.sln" --include=";**/*" --profile --profile=custom -dsl=GlobalAll -dsl=SolutionPersonal -dsl=ProjectPersonal
```

Luckily cleanupcode itself silently ignores the first profile parameter without a value and it uses the custom one.

Note that if your profile name contains spaces, you'll need to escape it:

```
dotnet regitlint --print-command --jb --profile --jb --profile="\"my custom profile name\""
```

which produces:

```
dotnet tool run jb cleanupcode "c:\Projects\demo.sln" --include=";**/*" --profile --profile="my custom profile name" -dsl=GlobalAll -dsl=SolutionPersonal -dsl=ProjectPersonal
```

if running on Windows using cmd.exe. But when running from PowerShell, different escaping for whitespace is needed:

```
pwsh dotnet regitlint --print-command --jb --profile --jb --profile='\"my custom profile name\"'
```

which produces:

```
dotnet tool run jb cleanupcode "c:\Projects\demo.sln" --include=";**/*" --profile --profile="my custom profile name" -dsl=GlobalAll -dsl=SolutionPersonal -dsl=ProjectPersonal
```

Aside from fixing this, it would be helpful to add an example to the docs that shows how to use profiles and how to escape spaces in the profile name.

**Comment by sethreno** [comment]
Thanks for the awesome bug report. I'll get this fixed up when I get a few minutes.

**Comment by vdurante** [comment]
Same happening here! Do you have any workaround?

**Comment by sethreno** [comment]
@vdurante bart mentions a workaround in his comment. 

> This results in cleanupcode to discard the first profile and use the second one. The workaround I found is to trick the code by passing --profile too:

    dotnet regitlint --print-command --jb --profile --jb --profile=custom

**Comment by sethreno** [comment]
This is fixed in 6.0.2
